src/components/NavBar/ChainSwitcher.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
what's this for?

Reply:
outdated and no longer needed, thanks for the catch